src/main/java/uk/gov/hmcts/reform/prl/services/barrister/BarristerRemoveService.java

```diff
@@ -27,13 +30,26 @@ public AllocatedBarrister getBarristerListToRemove(CaseData caseData, String aut
 
     @Override
     protected boolean isPartyApplicableForFiltering(boolean applicantOrRespondent, BarristerFilter barristerFilter, PartyDetails partyDetails) {
-        return hasBarrister(partyDetails) && partyDetails.getBarrister().getBarristerId() != null;
+        boolean isApplicable = hasBarrister(partyDetails) && partyDetails.getBarrister().getBarristerId() != null;
```

Comment:
I believe this implementation can be moved to Abstract class by passing in parameter isApplicable flag